code/game/mecha/equipment/weapons/mecha_ammo.dm

```diff
@@ -0,0 +1,94 @@
+/obj/item/mecha_ammo
+	name = "generic ammo box"
+	desc = "A box of ammo for an unknown weapon."
+	w_class = WEIGHT_CLASS_BULKY
+	icon = 'icons/mecha/mecha_ammo.dmi'
+	icon_state = "empty"
+	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
+	var/rounds = 0
+	var/round_term = "round"
+	var/direct_load //For weapons where we re-load the weapon itself rather than adding to the ammo storage.
+	var/load_audio = "sound/weapons/gun_magazine_insert_empty_1.ogg"
+	var/ammo_type
+
+/obj/item/mecha_ammo/proc/update_name()
+	if(!rounds)
+		name = "empty ammo box"
+		desc = "An exosuit ammuniton box that has since been emptied. Please recycle."
+		icon_state = "empty"
+
+/obj/item/mecha_ammo/attack_self(mob/user)
+	..()
+	if(rounds)
+		to_chat(user, "<span class='warning'>You cannot flatten the ammo box until it's empty!</span>")
+		return
+
+	to_chat(user, "<span class='notice'>You fold [src] flat.</span>")
+	var/I = new /obj/item/stack/sheet/metal(user.loc)
+	qdel(src)
+	user.put_in_hands(I)
+
+/obj/item/mecha_ammo/examine(mob/user)
+	..()
+	if(rounds)
+		to_chat(user, "There [rounds > 1?"are":"is"] [rounds] [round_term][rounds > 1?"s":""] left.")
```

Comment:
examine is now a list
```
. = ..()
. += "There [rounds > 1?"are":"is"] [rounds] [round_term][rounds > 1?"s":""] left."
```

Reply:
Doesn't work on my build, since it's behind, but I have it set now.